test/sshkit/ssh_test.exs

```diff
@@ -118,6 +143,15 @@ defmodule SSHKit.SSHTest do
       assert close(conn) == :ok
       assert_received :closed_sandbox_connection
     end
+
+    test "its logs the closed when dry_run is enabled" do
```

Comment:
In order to align this with the current naming conventions for tests in this file, I'd suggest to rename it to:
```elixir
test "log closing the connection when dry_run is enabled" do
```

*(for tiny changes like this, I'll just push to the PR branch myself)*